.github/workflows/release.yml

```diff
@@ -14,8 +14,28 @@ permissions:  # added using https://github.com/step-security/secure-workflows
   contents: read
 
 jobs:
-  build:
+  release-please:
+    permissions:
+      contents: write  # for google-github-actions/release-please-action to create release commit
+      pull-requests: write  # for google-github-actions/release-please-action to create release PR
     runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.container }}
```

Comment:
I don't think a container matrix is necessary for this job.

Reply:
Agreed, I've removed the container